tests/unit/components/Collections/ListItems/PlioListItem.spec.js

```diff
@@ -474,15 +476,15 @@ describe("PlioListItem.vue", () => {
 
   it("clicking analyze routes to Dashboard ", async () => {
     // set `matches` as `True` for testing on touch screen devices
-    setMatchMedia(true);
+    setMatchMedia(true); // deepscan-disable-line
```

Comment:
Does setting `setMatchMedia` as true means that if any of our tests later on will require testing on a non-touch screen device, we'll have to set it back again to false?
Wouldn't it be better if we create a separate describe for touch devices and a separate describe for non-touch devices?